sections/faqs/reverse-selectors.js

```diff
@@ -51,6 +51,25 @@ const ReverseSelectors = () => md`
 
   We could have nested the color-changing rule within our Link component, but then we'd have to
   consider both sets of rules to understand why Icon behaves as it does.
-`
 
-export default ReverseSelectors
+  ### Caveat
+
+  This behaviour is only supported within the context of *Styled* Components: attempting to mount 
+  `B` in the following example will fail (`Cannot call a class as a function`) because the referenced
```

Comment:
> (`Cannot call a class as a function`)

Maybe we can add another sentence saying something along the lines of:

> It will also throw an error saying "cannot call a class as a function," because the styled component will attempt to call the component as an interpolation function.